install/prepare_site_init.md

```diff
@@ -417,6 +418,60 @@ information and review the
 [Adding SNMP Credentials to the System](../operations/network/management_network/snmp_exporter_configs.md#adding-snmp-credentials-to-the-system)
 section for links to the relevant procedures.
 
+### Configure Max Istio Gateway Pods
+
+The Istio Gateways have a `Horizontal Pod Autoscaler` configured to allow Kubernetes to grow and shrink the
+deployment as needed due to increased load. This by default is configured to a max of 6 and a minimum of 3
+pods. On some smaller systems this leads to Kubernetes attempting to start up more pods than available
+worker nodes. This leads to test failures and a pod stuck in the pending state. The minimum and maximum
+should be configured to fit the system.
+
+This step can be skipped if the default of a maximum of 6 pods and a minimum of 3 pods is appropriate for
+the system size.
+
+1. (`pit#`) Set the minimum and maximum replicas wanted
+
+> ***NOTE*** the maximum should not be higher than the total number of worker nodes on a system, and
+the minimum has to be less than or equal to the maximum. This can have both be the same value but
+that will mean the pods will never automatically create or remove pods as load changes.
+
+```bash
+max_amount=4
+min_amount=3
+```
+
+1. (`pit#`) Update the customizations file.
+
+```bash
+yq write -i ${SITE_INIT}/customizations.yaml \
+  'spec.kubernetes.services.cray-istio-ingress.deployments.istio-ingressgateway.autoscaleMax' \
+  "$max_amount"
+yq write -i ${SITE_INIT}/customizations.yaml \
+  'spec.kubernetes.services.cray-istio-ingress.deployments.istio-ingressgateway.autoscaleMin' \
+  "$min_amount"
+
+yq write -i ${SITE_INIT}/customizations.yaml \
+  'spec.kubernetes.services.cray-istio-ingress.deployments.istio-ingressgateway-customer-admin.autoscaleMax' \
+  "$max_amount"
+yq write -i ${SITE_INIT}/customizations.yaml \
+  'spec.kubernetes.services.cray-istio-ingress.deployments.istio-ingressgateway-customer-admin.autoscaleMin' \
+  "$min_amount"
+
+yq write -i ${SITE_INIT}/customizations.yaml \
+  'spec.kubernetes.services.cray-istio-ingress.deployments.istio-ingressgateway-customer-user.autoscaleMax' \
+  "$max_amount"
+yq write -i ${SITE_INIT}/customizations.yaml \
+  'spec.kubernetes.services.cray-istio-ingress.deployments.istio-ingressgateway-customer-user.autoscaleMin' \
+  "$min_amount"
+
+yq write -i ${SITE_INIT}/customizations.yaml \
+  'spec.kubernetes.services.cray-istio-ingress.deployments.istio-ingressgateway-hmn.autoscaleMax' \
+  "$max_amount"
+yq write -i ${SITE_INIT}/customizations.yaml \
+  'spec.kubernetes.services.cray-istio-ingress.deployments.istio-ingressgateway-hmn.autoscaleMin' \
+  "$min_amount"
```

Comment:
This might be simpler to just put it in a loop, for example:
```suggestion
for gw in "" -customer-admin -customer-user -hmn; do
  yq write -i ${SITE_INIT}/customizations.yaml \
    'spec.kubernetes.services.cray-istio-ingress.deployments.istio-ingressgateway${gw}.autoscaleMax' \
    "$max_amount"
  yq write -i ${SITE_INIT}/customizations.yaml \
    'spec.kubernetes.services.cray-istio-ingress.deployments.istio-ingressgateway${gw}.autoscaleMin' \
    "$min_amount"
done
```